sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,53 @@
+# Basic device update enumerations
+
+This sample demonstrates basic operations with `DeviceUpdateClient` class in this library. `DeviceUpdateClient` is used to manage updates in Device Update for IoT Hub - each method call sends a request to the service's REST API.  To get started, you'll need a connection string to the Azure App Configuration. See the [README](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/deviceupdate/Azure.IoT.DeviceUpdate/README.md) for links and instructions.
+
+ ## Create a DeviceUpdateClient
+ 
+To interact with Device Update for IoT Hub, you need to instantiate a `DeviceUpdateClient`. You use either an endpoint URL, instance identity and a [`TokenCredential`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md#credentials).
+ 
+For the sample below, you can set `accountEndpoint` and `instance` in an environment variable.
+
+```C# Snippet:AzDeviceUpdateSample1_CreateDeviceUpdateClient
+var credentials = new DefaultAzureCredential();
+var client = new DeviceUpdateClient(accountEndpoint, instance, credentials);
+```
+
+## Enumerate all device update providers
+
+First, let's try to enumerate all available (already imported) device update providers.
+
+```C# Snippet:AzDeviceUpdateSample1_EnumerateProviders
```

Comment:
Where is this snippet from? I could not find it in test codes.

Reply:
I misunderstood how snippets work. I have removed the references to snippets. I would like to merge it now without them and then add the proper snippet references in the next version. Is that possible?

Reply:
That's OK